### ISSUE TITLE
Login method selection is aria-hidden

### ISSUE THREAD

**Issue by fcnjd** [hedgedoc/hedgedoc#2126]
### Description

I as a blind user, want to be able to choose the right login method with my screenreader, although the selection form is aria-hidden, making it hard to do that.

### Steps to reproduce

1. Launch a screenreader
2. On an instance which presents a login method selection, press the sign in button
3. You see that you don't find the selection

### Expected behaviour

The div element of the modal should not have the aria-hidden=true attribute, since this does nothing else than hiding the whole modal from screenreaders. I really don't know how it came there, but I consider it false.

### Your Setup

Tested on Windows 10

**Comment by mrdrogdrog** [comment]
The answer to this is pretty easy. The modals are always present in the DOM but hidden. That makes it easier to show them quickly. To make sure that screen readers don't see them they also get an aria-hidden attribute.
The problem: If the modals are shown then the aria-hidden attribute won't be removed. Should be pretty easy to fix.

**Comment by fcnjd** [comment]
Ah, thank you for your explanation, now I understand that. And good to know that it shouldn't be very hard to fix that.

**Comment by mrdrogdrog** [comment]
Fixed by https://github.com/hedgedoc/hedgedoc/pull/2242
